packages/components/src/components/toast/toast/toast-manager.ts

```diff
@@ -1,9 +1,11 @@
 import { forceUpdate } from '@stencil/core';
 import type { ToastState } from './toast-utils';
-import { ANIMATION_DURATION } from './toast-styles';
+import { motionDurationLong } from '@porsche-design-system/utilities-v2';
 import { throwException } from '../../../utils';
+import { cssVariableMotionDuration } from '../../../styles';
 
 const TOAST_DEFAULT_TIMEOUT = 6000;
+const MOTION_DURATION = parseFloat(motionDurationLong.replace(/(\d+\.?\d+)/, '$1')) * 1000;
```

Comment:
I think we don't need the replacement. 
<img width="153" alt="Bildschirmfoto 2023-11-06 um 09 07 11" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/651b4336-5572-40db-87de-bc4ec825e879">